spec/fixtures/sample.js

```diff
@@ -10,4 +10,20 @@ var quicksort = function () {
   };
 
   return sort(Array.apply(this, arguments));
-};
\ No newline at end of file
+};
+
+foo({
+    atIndentTwo: true
```

Comment:
>  I really don't see what's wrong with: ...

Can you give some examples of popular open source code that's formatted like that? It seems highly unusual to me.

I feel your concern regarding consistency, but it's important not to lose the forest for the trees here. The purpose of formatting code is to make it easier for people to read. In that regard, formatting  has more in common with natural language than it does with mathematics; familiarity is important, and fitting in with the style of the document you're editing is important.